packages/handleRichTextBoxKeyEventNavigation/package.json

```diff
@@ -7,13 +7,15 @@
     "author": "Quip",
     "main": "dist/handleRichTextBoxKeyEventNavigation.js",
     "scripts": {
-        "build": "cross-env-shell NODE_ENV=production webpack --output-library-target umd"
+        "build": "cross-env-shell NODE_ENV=production webpack"
```

Comment:
Why do we want to remove `--output-library-target umd`?

Reply:
so i added a newer version of `webpack` via the `webpack-cli` dependency. based on my research it seems that the `--output-library-target` argument has been deprecated. we already configure the output to be `umd` in our `webpack.config.js` files anyways so i didn't think it'd be an issue to remove the argument. local builds show that the files are indeed in the correct `umd` format and tests all pass so that gave me more confidence that the change should be safe

Reply:
Got it, thanks for the additional info!